base/setup/usetup/lang/it-IT.h

```diff
@@ -36,7 +36,7 @@ static MUI_ENTRY itITLanguagePageEntries[] =
     {
         8,
         13,
-        "\x07  Questa lingua sar� quella predefinita per il sistema finale.",
+        "\x07  Questa lingua sar� quella predefinita per il sistema finale.",
```

Comment:
Is that supposed to be encoded in UTF-8? If so, then you may want to revert back the changes. USETUP (text-mode setup) is prone to break special characters with UTF-8 encoding.